### PR TITLE
added sync binding to keycloak

### PR DESCRIPTION
**Pull request by cfoskin**
@wtrocki  Can you take a look and see if this makes sense?

### CODE REVIEW

File: modules/ROOT/pages/sync/binding_to_keycloak.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+* Ensure {sync-service} is xref:sync/provisioning.adoc[provisioned].
+* Ensure {keycloak-service} is xref:keycloak/provisioning.adoc[provisioned]
+
+== Creating the binding secret from a Keycloak instance
+
+* Navigate to your Openshift project
+* Select *Applications* and then *Provisioned Services*
+* Select the *Identity Management* service from the list of Provisioned Services
+* Select the *Create Binding* option
+* Enter the *Service Instance Id* of the provisioned *{sync-service}* service
```
Comment (wtrocki) [inline modules/ROOT/pages/sync/binding_to_keycloak.adoc]: Putting "newbie" hat > What is service instance ID?
 Maybe we can just say: Go to sync service section. Copy the Id.. etc. would it be possible?

**Comment by cfoskin** [comment]
screen shot 2018-08-17 at 15 10 09

**Comment by pmacko1** [comment]
@wtrocki will take a look re building. If it makes sense tech-wise, i can merge it then

**Comment by pmacko1** [comment]
hi, why do you want to merge it into sync, not master? thanks

**Comment by wtrocki** [comment]
Good to go! I got impression that master is for releases so let's stage that in sync branch

**Comment by pmacko1** [comment]
@wtrocki I'm not aware that we have mobile services versioned now, but  don't take my word for that. @finp 
Only master branch goes live, do you want to do that?

**Comment by wtrocki** [comment]
I mean that we going to merge when all will be ready.

**Comment by wtrocki** [comment]
@pmacko1 Can we merge this?

**Comment by wtrocki** [comment]
Actually we need that now as it's blocking us from writing more docs. - it goes to sync branch anyway

**Pull request merged**